src/context/update.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { https } from 'follow-redirects'
+import fs from 'fs'
+import { tmpNameSync } from 'tmp'
+import { commands, ProgressLocation, Uri, window } from 'vscode'
+import '../shared/extension'
+
+// Usage:
+// commands.registerCommand('sarif.updateExtension', async () => await update())
+export async function update() {
```

Comment:
>update [](start = 22, length = 6)

Are we removing automatic insider updates?

Reply:
Looks like I accidentally answered this question above. If you feel strongly about having this here, I'd be happy to remove it.

Reply:
It is back. Just pending some proxy work and real life testing.

---
In reply to: [430754296](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r430754296) [](ancestors = 430754296)